lib/config/configuration.go

```diff
@@ -118,6 +122,30 @@ func ReadConfigFile(cliConfigPath string) (*FileConfig, error) {
 	return ReadFromFile(configFilePath)
 }
 
+// ReadResources loads a set of resources from a file.
+func ReadResources(filePath string) ([]services.Resource, error) {
+	reader, err := utils.OpenFile(filePath)
+	defer reader.Close()
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	decoder := kyaml.NewYAMLOrJSONDecoder(reader, 32*1024)
```

Comment:
Can you use `utils.ReadYAML` instead? Also can you make `32*1024` a constant?

Reply:
I can definitely make `32*1024` a constant.  That value is used everywhere that `NewYAMLOrJSONDecoder` is called, where would be the appropriate place to put a commonly used constant like that?

In terms of using `utils.ReadYAML`, I'm happy to, but I'm not sure how lol (golang newbie problems).  From what I can tell, `utils.ReadYAML` dynamically creates everything as a bunch of primitive values (maps,strings,etc).  Does golang have some kind of mechanism for attempting to convert a composite of primitives to a concrete type?